framework/test/jest/integration/specs/modules/chain/genesis_block.spec.js

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const {
+	createDefaultLoadedChainModule,
+	StorageSandbox,
+	storageConfig,
+	getBlock,
+	getAccount,
+	getDelegateList,
+} = require('../../../utils');
+const delegateListForTheFirstRound = require('../../../../../fixtures/config/devnet/round_delegates_for_first_round.json');
+const genesisBlock = require('../../../../../fixtures/config/devnet/genesis_block');
+
+describe('genesis block', () => {
+	const dbName = 'genesis_block';
+	const TRANSACTION_TYPES_DELEGATE_REGISTRATION = 2;
+	let storage;
+	let chainModule;
+
+	beforeAll(async () => {
+		storage = new StorageSandbox(storageConfig(dbName));
```

Comment:
can we call storageConfig inside StorageSandbox and just provide the config object to storageSandbox ?